prov/gni/src/gnix_wait.c

```diff
@@ -195,17 +347,13 @@ static int gnix_init_wait_obj(struct gnix_fid_wait *wait, enum fi_wait_obj type)
 	wait->type = type;
 
 	switch (type) {
-	case FI_WAIT_FD:
```

Comment:
If we don't support WAIT_FD, why did we use the behavior defined below prior to this mod? Can you add comments to either the code, or your commit to explain this better.